### PR TITLE
Not running installers will not disable auto-subscribe

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: nservicebus/operations/installers.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ The alternative to using installers is to create the required resources before t
 
 partial: installer-api
 
+## What is not part of installers
```
Comment (ramonsmits) [inline nservicebus/operations/installers.md:22]: Maybe this should be "Auto subscribe is not part of installers"?

Comment (jpalac) [inline nservicebus/operations/installers.md:22]:
```suggestion
## Auto-subscribe is not part of installers
```

File: nservicebus/operations/installers.md [hunk 2]
```diff
@@ -19,6 +19,10 @@ The alternative to using installers is to create the required resources before t
...
+
+NServiceBus detects all events an endpoint handles and [auto subscribes to these events at startup](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#automatic-subscriptions). Auto subscribing to events can be disabled by [disabling the auto subscribe feature](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#disabling-auto-subscription).
```
Comment (jpalac) [inline nservicebus/operations/installers.md:24]:
```suggestion
NServiceBus detects all events an endpoint handles and [auto-subscribes to these events at startup](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#automatic-subscriptions). Automatic subscriptions can be turned off by [disabling the auto subscribe feature](/nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md#disabling-auto-subscription).
```

**Review by jpalac — COMMENTED** [review]
Couple of small changes

**Pull request merged**